asciidoc/courses/aura-dashboards/modules/1-getting-started/lessons/3-exploring-dashboards/lesson.adoc

```diff
@@ -0,0 +1,148 @@
+= Exploring Dashboards
+:type: lesson
+:order: 3
+
+== Load data into your Aura instance
+
+To create dashboards in Neo4j Aura, you will need to:
+
+* Connect to a Neo4j database - either Cloud Aura instance or local instance.
+* Either upload a CSV file - from local system or from Cloud storage, this requires knowledge of how to Import data.
+* OR Run the *import-recommendations.cypher* script provided in this lesson, this will load all the necessary Movies Recommendation dataset nodes and relationships.
+* Map properties from the CSV file to the node
+* Run the import
+
+
+
+=== Step 1: Connect to your database
+To connect to your Neo4j Aura instance, open the Import Data tool from the Aura interface.
+
+Use the connection details of your Aura instance to connect.
+
+image::images/05-inspect-details.png[Inspecting connection details of an Aura instance]
+
+
+If you have any existing on-prem Neo4j database, you can also connect to it by entering the connection details.
+
+
+If you have an existing data importer model, you can clear it by selecting the `Clear all` option from the menu `...`.
+
+image::images/clear-model.png[The clear all option in the data importer menu '...']
+
+
+Any data import you run will be against the connected Neo4j database.
+
+=== Step 2: Navigate to the Query Editor
+
+After connecting to your Aura instance, you will need to navigate to the Query Editor in order to load data using Cypher.
+
+At this point, you do not need to understand Cypher in detail; you will only run the script that loads the data.
+
+image::images/query-menu.png[The query menu in Aura Dashboards]
+
+=== Step 3: Import the Movies Recommendation dataset
+
+Use the following Cypher script to create the Movies Recommendation dataset in your Aura instance:
+
+link:https://github.com/neo4j-graphacademy/recommendations-extend/blob/main/recommendations-extended/data/cypher/import-recommendations.cypher[Download the import-recommendations.cypher script^]
+
+You can copy and paste the script into the Query Editor, and click the **Run** button to execute it.
+
+
+image::images/run-query.png[The import-recommendations.cypher script in the Query Editor]
+
+Once the script has finished running, you will see a summary of the nodes and relationships created in your Aura instance. 
+
+Go to the Explore tab to visualize and explore the data you have just loaded: 
```

Comment:
An easier way to quickly see the imported data is in Query by running `CALL db.schema.visualization`
In Explore, you need search for something or use the _Show a graph snippet_ button in the empty state.